addons/mail/static/src/models/thread/thread.js

```diff
@@ -732,6 +795,45 @@ function factory(dependencies) {
             return this.messaging.currentPartner.moderatedChannelIds.includes(this.id);
         }
 
+        /**
+         * @private
+         * @returns {mail.message}
+         */
+        _computeLastCurrentPartnerMessageSeenByEveryone() {
+            if (!this.partnerSeenInfos || !this.mainCacheOrderedMessages) {
+                return [['unlink-all']];
+            }
+            const otherPartnerSeenInfos =
+                this.partnerSeenInfos.filter(partnerSeenInfo =>
+                    partnerSeenInfo.partner !== this.messagingCurrentPartner);
```

Comment:
many small lint issues such as ) close on new line, not sure if worth the time mentioning them

Reply:
nah it's fine... we'll apply prettier tomorrow, right?

Reply:
Assuming we have time, and assuming we are happy with the config/result.